packages/neuron-ui/src/widgets/Dropdown/index.tsx

```diff
@@ -68,7 +68,7 @@ export const CustomizableDropdown = ({
   )
   return (
     <div className={`${styles.customizableRoot} ${className || ''}`}>
-      {children}
+      <div>{children}</div>
```

Comment:
Is wrapping `children` in `div` related to the upgrade of react?

Reply:
> Is wrapping `children` in `div` related to the upgrade of react?

Yes, it is related to the upgrade of react 18, in React 17 it's necessary. I'll change it back.